cl/lib/command_utils.py

```diff
@@ -17,6 +17,10 @@ def handle(self, *args, **options):
             logger.setLevel(logging.INFO)
         elif verbosity > 1:
             logger.setLevel(logging.DEBUG)
+            # This will make juriscraper's logger, called "Logger"
+            # accept most logger calls.
+            juriscraper_logger = logging.getLogger("Logger")
```

Comment:
Should we rename this with something better?

Reply:
The naming convention for loggers it to define them like this `logging.getLogger(__name__)`, which gives them the module name, but the change is done on juriscraper [first](https://github.com/freelawproject/juriscraper/blob/main/juriscraper/lib/log_tools.py), and then here

I think it shouldn't have any side effect, so let's update it

Reply:
This is corrected now, including the juriscraper side, please check again @mlissner